rpm/parted.spec

```diff
@@ -8,7 +8,7 @@ Version: 0
 Release: 1
 License: GPLv3+
 Group:   Applications/System
```

Comment:
Bonus points if you'd drop Group as well [1]

[1] https://docs.fedoraproject.org/en-US/packaging-guidelines/